yacl/math/mpint/tommath_ext_features.cc

```diff
@@ -182,11 +182,11 @@ void mp_ext_safe_prime_rand(mp_int *p, int t, int psize) {
     // Find a odd number `q` among q, q+2, .... , (1 << 20) satisfy：
     // 1. co-prime to `small_primes`.
     // 2. `q = 1 mod 3` (p = 2q+1).
-    MPINT_ENFORCE_OK(mp_mod_d(&q, small_prime_prod, &mod));
+    MPINT_ENFORCE_OK(mp_mod_d(&q, small_prime_prod, &original_m));
```

Comment:
The variable mod will be used later for other usage.

Reply:
nice catch~